src/commonMain/kotlin/core/Either.kt

```diff
@@ -1,4 +1,4 @@
-package io.kotest.assertions.arrow.core
+package core
```

Comment:
Should the FQN just be core here ?

Reply:
ups, gonna change that. I missed it in the refactor